network/manager.go

```diff
@@ -0,0 +1,569 @@
+package network
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/nknorg/nconnect/admin"
+	"github.com/nknorg/nconnect/config"
+	"github.com/nknorg/nkn-sdk-go"
+)
+
+const (
+	defaultDomain  = "nconnect.nkn"
+	defaultIpStart = "192.168.0.1"

```

Comment:
I wouldn't use 192.168 by default, this IP range conflicts with too many routers. I would use 10.X.Y range with a rarely used X and Y

Reply:
Yes, modified. It is configurable on the manager's web page too.